osu.Game/Rulesets/Mods/ModTimeRamp.cs

```diff
@@ -69,6 +72,7 @@ public virtual void Update(Playfield playfield)
         /// </summary>
         /// <param name="amount">The amount of adjustment to apply (from 0..1).</param>
         private void applyAdjustment(double amount) =>
-            SpeedChange.Value = 1 + (FinalRate.Value - 1) * Math.Clamp(amount, 0, 1);
+            //  SpeedChange.Value = 1 + (FinalRate.Value - 1) * Math.Clamp(amount, 0, 1);
```

Comment:
Commented code shouldn't stay in if no longer needed.

Reply:
whoops my bad